src/ProjectDataProvider.ts

```diff
@@ -325,11 +325,6 @@ export module ProjectDataProvider {
       }
 
       const cmd: string = [
-        Config.getGoExecutable(),
-        `get`,
```

Comment:
The problem seems to be with the go run itself and the issue shows how it is not working.

Reply:
I have some improvements by setting GOPATH to temporary directory. I will share extension for you to verify it once.